.travis.yml

```diff
@@ -43,9 +43,6 @@ env:
     - secure: "eVA62wnIJW4/hZ1OCDQCAw+MvbvNy18yN5LkVtFz/ZEB4Y8k5BJz1RNWPrO7ymOUA3O4wcqVjiMlEd6JgTvsuKX/Q5acLw66RO0NUEovRaC7hunTMBxby/kQs8jR7238yC2GAhTSN0bIqnhwejSb+ERN/J7hx8dack6VJppoJ4sIkYtMzTZOPcVfvdxAa5NCzZCIqZTJKOfXX3srFhNaadCq9FQLDGHyehMqgA0eDUf+WuwXPJ/a4oWTkucHl8XQTxSb0cqxSrAX8MHpINU/UcHu7l9/lE2MWS08bQVS+FPp58PldsNtjNDFN7f/0g30XYs+xCNX9fGxRlTxPD5pvB32WeWT+xxzQCdkku9QGch7/RY4g8+5rEIfsm/aMVKai9CaLJJJDTMnXBfhSSPXNRWfbDg7pOae0kdz4+Y+Q2hHcQHDdBBzQBZ7458ujImsXbqS3jhWE5Ogh88jTw9iUiYdPDTFO0TR5JLXG6k6x7OrmurDe5SxH29NPHJl+N8KjH9XHWTVj4mKJPOLDLzwPzBvZsEr8DfB07+2PtooHMLedy9jE65sOOkwVywO7bPFtk2GtpHFYLOzCtnN9aGw3OWni46Q1vVYIf47k6bO2rn2yK/rCTRUEGTCcddWcbUQFtteFe8B5cajShdZIXoQ3C6bl+JbuDzOSHHGECuBgaI="
 
   matrix:
-    # what we used for the latest PMM release
```

Comment:
That comment was accurate for 1.14, please keep it.

Please also add "tip" version